tutorials/buoyantSimpleFoam-aster/precice-config.xml

```diff
@@ -31,6 +31,7 @@
       <read-data mesh="Solid-Interface-Faces" name="Sink-Temperature-Fluid"/>
       <read-data mesh="Solid-Interface-Faces" name="Heat-Transfer-Coefficient-Fluid"/>
       <mapping:nearest-neighbor constraint="consistent" direction="read" to="Solid-Interface-Faces" from="Fluid-Interface"/>
+       <export:vtk directory="preCICE-output" />
```

Comment:
This is an additional setting, not related to preCICE v2. I would suggest that we remove it, for consistency with other tutorials.

```suggestion
```

Reply:
Solved